source/changelog.txt

```diff
@@ -12,6 +12,32 @@ Release Notes
    :depth: 1
    :class: singlecol
 
+v1.7.1
+------
+
+*Released February 16, 2023*
+
+- :issue:`MONGOSH-1087` An earlier enhancement to autocomplete that is
+  available now.
```

Comment:
Apart from the ticket having been re-opened in the meantime, I really wouldn’t list it here since it implies that it is related to the 1.7.1 release, which it’s not.

(I looked it up, and the work that is being referenced here ended up in mongosh 1.6.0.)